build.sbt

```diff
@@ -20,17 +21,58 @@ lazy val microservice =
     .settings(Compile / unmanagedResourceDirectories += baseDirectory.value / "resources")
     .configs(IntegrationTest)
     .settings(inConfig(IntegrationTest)(Defaults.itSettings) *)
+    .settings(
+      IntegrationTest / Keys.fork := false,
+      IntegrationTest / unmanagedSourceDirectories := (IntegrationTest / baseDirectory)(base => Seq(base / "test")).value,
+      IntegrationTest / testOptions := Seq(Tests.Filter((name: String) => name startsWith "it")),
+      addTestReportOption(IntegrationTest, "int-test-reports"),
+      IntegrationTest / testGrouping := oneForkedJvmPerTest((IntegrationTest / definedTests).value),
+      IntegrationTest / parallelExecution := false,
+      // Disable default sbt Test options (might change with new versions of bootstrap)
+      IntegrationTest / testOptions -= Tests
+        .Argument("-o", "-u", "target/int-test-reports", "-h", "target/int-test-reports/html-report"),
+      IntegrationTest / testOptions += Tests.Argument(
+        TestFrameworks.ScalaTest,
+        "-oNCHPQR",
+        "-u",
+        "target/int-test-reports",
+        "-h",
+        "target/int-test-reports/html-report")
+    )
```

Comment:
1. Oh, so the tests have reappeared. The history is unclean.
2. Could we write a function to add these options to all configs? Maybe create a common variable with test options?

Reply:
Maybe, I thought we could keep it simple for now since we are still trying to figure out how to get it to work properly for the it test config